dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/MailMessageCreator.java

```diff
@@ -261,7 +261,7 @@ private String dataverseMessage(EmailNotificationDto notificationDto, Dataverse
                                                                                         parentDataverse != null ? parentDataverse.getDisplayName() : "",
                                                                                         parentDataverse != null ? getDataverseLink(
                                                                                                 parentDataverse) : "",
-                                                                                        systemConfig.getGuidesBaseUrl(),
+                                                                                        systemConfig.getGuidesBaseUrl(session.getLocale()),
```

Comment:
Also here there can be no `SessionScope`.
It will be better to do: `notificationDto.getNotificationReceiver().getNotificationsLanguage()`
As we would point to guide in language that is the same as email content.